docs/contributing/assets/allowed_actions.json

```diff
@@ -4,7 +4,10 @@
       "name": "golangci/golangci-lint-action",
       "versions": [
         "639cd343e1d3b897ff35927a75193d57cfcba299",
-        "v3"
+        "v3",
+        "3cfe3a4abbb849e10058ce4af15d205b6da42804",
+        "v4",
```

Comment:
Remove this entry because it allows all minor versions of v4. If not really necessary I am also in favor of removing the next line and only use the commit hash of the release.

Reply:
I've removed `v4`. I'd like to keep `v4.0.0` to allow dependabot to bump it in the future